bundle.proto

```diff
@@ -14,3 +14,41 @@ message BundleUuid {
   bundle.Bundle bundle = 1;
   string uuid = 2;
 }
+
+/* Bundle Result Types */
+
+// Indicates the bundle was forwarded to validators.
+message Forwarded {
+  // Maps slots to validator pubkeys this bundle was forwarded to.
+  map<uint64, string> validator_identities = 1;
+}
+
+// Indicates the bundle was dropped and therefore not forwarded to any validator.
+message Dropped {
+  oneof reason {
+    AuctionFailure auction_failure = 1;
+    SimulationFailure simulation_failure = 2;
+    InternalError internal_error = 3;
+  }
+}
+
+
+message AuctionFailure {
```

Comment:
im getting anal on naming lol

wdyt about like AuctionDropped or something like that? it might have succeeded but didn't get included?

Reply:
naming apis are hard